lib/resources/example_json/soarian_examples_observation.rb

```diff
@@ -0,0 +1,161 @@
+module Cerner
+  module Resources
+
+    SOARIAN_OBSERVATION_BUNDLE ||= 
+    
+   {
+    "resourceType": "Bundle",
+    "id": "44900955-4315-4c77-a85e-8a8800677d3f",
+    "meta": {
+        "lastUpdated": "2017-05-30T02:44:10.364-04:00"
+    },
+    "extension": [
+        {
+            "url": "http://cerner.hs.fhir.com/StructureDefinition/bundle-result-messages",
+            "valueExtension": {
+                "url": "http://cerner.hs.fhir.com/StructureDefinition/bundle-result-messages",
+                "extension": [
+                    {
+                        "url": "Severity",
+                        "valueString": "info"
+                    },
+                    {
+                        "url": "Code",
+                        "valueString": "papi029"
+                    },
+                    {
+                        "url": "Message",
+                        "valueString": "Information provided is from 2017-05-01 through 2017-05-30 inclusive."
+                    }
+                ]
+            }
+        }
+    ],
+    "type": "searchset",
+    "link": [
+        {
+            "relation": "self",
+            "url": "base-url/bf615549-44cc-4899-bfa9-2cfd86bf358c/Observation?patientId=FCC941D7-60B9-491D-BEED-27629E47CD4E&_format=json"
+        }
+    ],
+    "entry": [
+        {
+            "resource": {
+                "resourceType": "Observation",
+                "text": {
+                    "status": "additional",
+                    "div": "&lt;ul ccdRefId=\"ZSocial_History.Social_History.121415.91005.value.0\">&lt;li>Tobacco Use? : Light Tobacco Smoker&lt;/li>&lt;/ul>"
+                },
+                "identifier": [
+                    {
+                        "use": "official",
+                        "value": "ZSocial_History.Social_History.121415.91005.value.0"
+                    }
+                ],
+                "status": "final",
+                "code": {
+                    "coding": [
+                        {
+                            "system": "http://loinc.org",
+                            "code": "72166-2",
+                            "display": "A_Tobacco?"
```

Comment:
@daliboz, Soarian does not have a master file of loinc codes with their loinc-assigned names.  We take the display name from the lab interface or service master.  At this point it is too late to add such a table and associated logic - we would miss certification deadlines.  Ask that we leave this as is rather than risk missing certification as systems should use the code for semantic interoperability and the text for display purposes.